qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -563,45 +567,56 @@ def generate_hamiltonian(
     return convert_observable(h_of), nr_qubits
 
 
-def sd_excitations(n_electrons, n_orbitals, delta_sz=0):
-    r"""Generates single and double excitations from a Hartree-Fock (HF) reference state.
+def excitations(n_electrons, n_spin_orbitals, delta_sz=0):
+    r"""Generates single and double excitations from a Hartree-Fock reference state.
 
-    The singly- and doubly-excited configurations are generated by acting with the operators
-    :math:`\hat T_1` and :math:`\hat T_2` on the HF state:
+    Single and double excitations can be generated by acting with the excitation operators
+    :math:`\hat T_1` and :math:`\hat T_2` on the Hartree-Fock (HF) reference state:
 
     .. math:
-        && \vert \Phi_\mathrm{S} \rangle = \hat{T}_1 \vert \mathrm{HF} \rangle = \sum_{r \in
-        \mathrm{occ} \\ p \in \mathrm{virt}} \hat{c}_p^\dagger \hat{c}_r \vert \mathrm{HF}
-        \rangle \\
-        && \vert \Phi_\mathrm{D} \rangle = \hat{T}_2 \vert \mathrm{HF} \rangle = \sum_{r>s \in
-        \mathrm{occ} \\ p>q \in \mathrm{virt}} \hat{c}_p^\dagger \hat{c}_q^\dagger
-        \hat{c}_r \hat{c}_s \vert \mathrm{HF} \rangle
+
+        && \hat{T}_1 \vert \mathrm{HF} \rangle = \sum_{r \in \mathrm{occ} \\ p \in \mathrm{virt}}
+        \hat{c}_p^\dagger \hat{c}_r \vert \mathrm{HF} \rangle \\
+        && \hat{T}_2 \vert \mathrm{HF} \rangle = \sum_{r>s \in \mathrm{occ} \\ p>q \in
+        \mathrm{virt}} \hat{c}_p^\dagger \hat{c}_q^\dagger \hat{c}_r \hat{c}_s
+        \vert \mathrm{HF} \rangle,
+
 
 	where the indices :math:`r, s` and :math:`p, q` run over the occupied (occ) and unoccupied,
```

Comment:
```suggestion
    where the indices :math:`r, s` and :math:`p, q` run over the occupied (occ) and unoccupied,
```
changing to 4 spaces over tabs, otherwise had problems viewing docs